axum-extra/src/multipart_builder.rs

```diff
@@ -0,0 +1,337 @@
+//! Generate forms to use in responses. You're probably looking for [MultipartForm].
+
+use axum::response::{IntoResponse, Response};
+use http::{header, HeaderMap};
+use std::time::{SystemTime, UNIX_EPOCH};
+
+/// The `Content-Transfer-Encoding` setting for a part.
+#[derive(Debug)]
+pub enum TransferEncoding {
+    /// If not specified, encoding defaults to UTF-8
+    Default,
```

Comment:
Defaults by us choosing it? You're setting it explicitly in the `Part::text` for example.

So I think this should be named `TextUtf8` or something se more descriptive.

If you want to signal somewhere you're using just a default that might change one day, derive `Default` and use that.